core/trino-spi/src/main/java/io/trino/spi/ptf/AbstractConnectorTableFunction.java

```diff
@@ -21,6 +22,7 @@
 
 import static java.util.Objects.requireNonNull;
 
+@Experimental(until = "2022-10-31")
```

Comment:
Any reason why this particular date? or is this just an example?

also, would it be good to have a link to a ticket describing the work stream that given experimental API belongs to?

Reply:
This is just and example... I figured 3 months was plenty of time to finish up any work on the initial SPI.

As for issue, can you propose something specific?